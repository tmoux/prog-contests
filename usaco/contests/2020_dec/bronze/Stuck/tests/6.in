30
E 386 100
E 726 994
E 552 578
N 946 290
N 970 51
E 631 593
N 627 312
E 214 355
E 90 412
N 610 969
N 641 620
N 987 888
E 566 770
E 856 417
E 260 849
N 205 59
N 518 945
N 873 458
N 637 289
N 607 478
N 314 471
N 100 459
E 438 25
E 74 233
N 681 493
E 270 699
N 839 569
N 622 794
N 847 431
E 682 390
