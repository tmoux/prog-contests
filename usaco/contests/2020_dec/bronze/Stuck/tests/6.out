584
261
129
127
366
208
266
413
537
Infinity
150
Infinity
273
Infinity
362
174
Infinity
536
289
100
228
Infinity
Infinity
896
100
371
425
Infinity
563
264
